### PR TITLE
naming class BEM

### PR DESCRIPTION
**Pull request by RVdub**

### CODE REVIEW

**Comment by RVdub** [comment]
Dear Etienne,
Are-you agree with this new version of index.html and enchantee.html ?
Hear from you soon
Harvey

File: enchantee.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+<body class="page">
+    <div class="page__wrapper">
+        <header class="header">
+            <a href="./index.html" class="header__link">
+                <h1 class="header__logo">ohmyfood</h1>
+            </a>
+        </header>
+
+        <main class="main">
+            <figcaption class="carte__caption">La note enchantée</figcaption>
```
Comment (Boyman10) [inline enchantee.html:37]: Where is the carte declared ? you might replace main by carte.

File: index.html [hunk 2]
```diff
@@ -24,59 +24,61 @@
...
-                        <a href="./menu_enchantee.html">
-                            <i class="fas fa-music"></i>
-                            <p class="nav__enchantee--heading">La note enchantée</p>
-                            <p class="nav__enchantee--byline">MENU</p>
+        <main class="main">
+            <nav class="main__nav">
+                <ul class="menu">
+                    <li class="menu__enchantee">
+                        <a href="./enchantee.html" class="menu__link">
+                            <i class="fas fa-music" class="menu__enchantee--icon"></i>
```
Comment (Boyman10) [inline index.html:40]: icon isn't a modifier :)

**Comment by RVdub** [comment]
All text class in englsh
revocation of this pull request

**Review by Boyman10 — COMMENTED** [review]
Review

**Pull request closed without merging**